pyerrors/input/sfcf.py

```diff
@@ -236,6 +236,10 @@ def read_sfcf_multi(path, prefix, name_list, quarks_list=['.*'], corr_type_list=
             rep_path = path + '/' + item
             if "files" in kwargs:
                 files = kwargs.get("files")
+                if isinstance(files, list):
+                    if isinstance(files[0], list):
+                        files = files[i]
+
```

Comment:
What happens if files in to of type list[list[int]]? Maybe we should add an explicit exception here or is there a type check somewhere else that I overlooked?

Reply:
Hi, sorry it took some time, I put explicit type checks in both places where `kwargs.get('files')` is used and just put two tests.